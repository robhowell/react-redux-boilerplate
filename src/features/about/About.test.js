import React from 'react';
import { shallow } from 'enzyme';
import About from './About';

describe('Features/About Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
