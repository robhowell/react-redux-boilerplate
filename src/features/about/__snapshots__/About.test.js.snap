// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features/About Component should render correctly 1`] = `
<div>
  <h2>
    About Page
  </h2>
</div>
`;
